prov/efa/src/rdm/efa_rdm_util.h

```diff
@@ -8,7 +8,11 @@
 #include "efa_rdm_protocol.h"
 #include "efa_rdm_pke.h"
 
-#define EFA_RDM_MSG_PREFIX_SIZE (sizeof(struct efa_rdm_pke) + sizeof(struct efa_rdm_eager_msgrtm_hdr) + EFA_RDM_REQ_OPT_RAW_ADDR_HDR_SIZE)
+#define EFA_RDM_MSG_PREFIX_SIZE (			\
```

Comment:
We need to update this protocol doc https://github.com/ofiwg/libfabric/blob/main/prov/efa/docs/efa_rdm_protocol_v4.md#43-keep-packet-header-length-constant-constant-header-length-and-zero-copy-receive for this.

Reply:
I am slightly concerned whether this will break backward compatibility. But I think it's fine as long as the sender & receiver use the same prefix size. Using a smaller prefix size on receiver will break, but we should be able to have a way to prevent this from happening.

Reply:
I mean, this is NOT a protocol breaking change.